src/components/Login.tsx

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import { useState } from 'react';
+
+import { Form, FormSection } from 'src/components/Form';
+import { Button, InlineFeedback, TextInput } from 'src/components/ui';
+import { useToast } from 'src/context/toast';
+
+export default function Login() {
+  const [email, setEmail] = useState<string>('');
+  const [password, setPassword] = useState<string>('');
+  const [isEmailValid, setIsEmailValid] = useState<boolean>(false);
+  const [isPasswordValid, setIsPasswordValid] = useState<boolean>(false);
+  const [shouldShowErrorState, setShouldShowErrorState] = useState(false);
```

Comment:
```suggestion
  const [shouldShowError, setShouldShowError] = useState(false);
```

Since it's already a state variable, we don't need to explicitly mention state in the variable name

Reply:
Thanks for the catching! Updated.